governance/GOVERNANCE.md

```diff
@@ -48,6 +48,14 @@ cannot veto or remove items.
 The moderator is responsible for summarizing the discussion of each
 agenda item and sends it as a pull request after the meeting.
 
+### Guidelines for sharing Board information in CPC meetings
+To improve organizational transparency and collaboration between the CPC and Board, it is desireable for CPC Directors to be able update the CPC on topics discussed during Board meetings. The goal of the below guidelines is to make this process lightweight and allow for information to be shared with the CPC as freely as possible. This is why the audience of those Board updates are somewhat restricted.
+
+- CPC Directors should use their own judgement about what may be shared with the CPC. When in doubt they should abstain from sharing information and seek epxlicit permission to do so.
+- In particular, the following information must never be shared with the CPC: information pertaining to a specific member, to staff, or about legal matters.
```

Comment:
Maybe be more specific about this? Or some kind of wording that would be similar to "good judgement"?

Reply:
Could you suggest a change? As I'm not sure I understand what you mean.

Reply:
^ @ovflowd

Reply:
Something in line with "Use your good judgement when sharing information with the CPC; Information that is sensitive, or regarding legal matters are examples of what shouldn't be shared"